src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -140,7 +142,7 @@ private double inputProcessing(double value) {
 	 * @param axis The processed axis
 	 * @return The processed value.
 	 */
-	private double ProcessedAxisValue(GenericHID hid, Axis axis){
+	private double processedAxisValue(GenericHID hid, Axis axis){
```

Comment:
why is it uncapitalized you monster

Reply:
umm smth charles told me to do a while ago

Reply:
but ok ill recapitalize it